interface/super/edit_layout.php

```diff
@@ -57,6 +57,9 @@
   "2"  => xl("Textbox"),
   "3"  => xl("Textarea"),
   "4"  => xl("Text-date"),
+  "5"  => xl("Email"),
+  "6"  => xl("Numeric"),  
```

Comment:
Just curious.  If it is integer only, I would recommend calling it that vs. Numeric.  Float/double/exponent etc... is sort of what some of us think as inclusive in "numeric".